pre_commit/resources/hook-tmpl

```diff
@@ -7,6 +7,7 @@ HERE=`pwd`
 popd > /dev/null
 
```

Comment:
since we're changing this file, we'll need to bump the magical number at the top.  I usually just md5 setup.py and put that there (you'll need to update the list of these as well)